webpack/assets/javascripts/react_app/components/DiffView/__snapshots__/DiffView.test.js.snap

```diff
@@ -2,8 +2,11 @@
 
 exports[`DiffView rendering render DiffView w/Patch 1`] = `
 <div>
-  <Memo(Diff)
+  <Diff
     diffType="modify"
+    generateAnchorID={[Function]}
+    gutterAnchor={false}
+    hideGutter={false}
     hunks={
```

Comment:
This makes the tests fail and is most definitely irrelevant, can we reverse the change to this file? :)